android/documentation/view_creation.md

```diff
@@ -0,0 +1,137 @@
+	
+		<Button
+			android:id="@+id/login_button"
+			android:text="@string/sign_in" />
+	
+	</com.your.package.LoginFullView>
+	```
+	
+2. Create the new custom view class used in the layout XML. In this example, this is `LoginFullView`. As before, you can use the default `LoginView` class as a template. In the new class, first get a reference to the components. The components in this example are the Add button and its listener. The class must also implement the `LoginViewModel` interface, with its getter and setter methods. In this case, the `getPassword()` method returns the `ANDROID_ID`.
+
+3. Insert the `LoginScreenlet` in any of your activities or fragments, using `@layout/login_full` as the `liferay:layoutId` attribute's value.
+
+## Child View
+
+The example Child view here presents the same components as the Default view, but uses a completely different layout: the view is oriented horizontally instead of vertically. The following steps show how to create a Child view.
+
+1. Create a new layout file called `login_horizontal.xml`. This is where the new UI is built. A good way to start is to duplicate the parent layout's XML and use it as a template. The child components such as the EditText fields, Buttons, and so on must use the same identifiers as the parent view. You must also use the same custom view class as the parent view. The example `login_horizontal.xml` is shown here: 
+
+	```xml 
+	<?xml version="1.0" encoding="utf-8"?>
+	<com.liferay.mobile.screens.themes.default.auth.login.LoginView
+		xmlns:android="http://schemas.android.com/apk/res/android"
+	    style="@style/default_screenlet"
+		android:orientation="horizontal">
+	
+	    <EditText
+		android:id="@+id/login"
+		style="@style/default_edit_text"
+		android:drawableLeft="@drawable/default_mail_icon"
+		android:hint="@string/email_address"
+		android:inputType="text" />
+	
+	    <EditText
+		android:id="@+id/password"
+		style="@style/default_edit_text"
+		android:drawableLeft="@drawable/default_lock_icon"
+		android:hint="@string/password"
+		android:inputType="textPassword" />
+	
+	    <Button
+		android:id="@+id/login_button"
+		style="@style/default_submit_button"
+		android:text="@string/sign_in" />
+	
+	</com.liferay.mobile.screens.themes.auth.login.LoginScreenletView>
+	```
+
+2. Insert the `LoginScreenlet` in any of your activities or fragments, and use `@layout/login_horizontal` as the `liferay:layoutId` attribute's value.
+
+## Extended View
+
+The example Extended view here presents the same components as the Default view, but adds new functionality to the screenlet: the password strength is computed before sending the request.
+
+1. Create a new layout file called `login_password.xml`. However, there's no need to actually build a new UI. All you need to do is change the parent's class to a new custom class. To use the same UI, use the `<include>` tag to import the parent's layout:
+
+	```xml 
+	<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I was thinking about another solution, but I realized that it is not gonna work anyway :)